canonical-web-and-design/ubuntu.com
# 3133 new pagination on resources

## Done

- Implemented blog style pagination on /resources
- Changed links on /resources from blog.ubuntu.com to u.c/blog
- Moved pagination template to shared template directory

## QA

- Check out this feature branch
- Run the site using the command `./run`
- View the site locally in your web browser at: [/resources](http://0.0.0.0:8001/resources)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Scroll to the pagination at the bottom of the page, click any of the links there, and see that you are presented with another page worth of posts
- Click any of the posts, see that you are taken to the post on u.c, not blog.ubuntu.com


## Issue / Card

Fixes #3133
